end-to-end-tests/cypress/support/commands.js

```diff
@@ -39,21 +39,19 @@ Cypress.Commands.add("navToEditorByDrawer", (viewerLinkText, navigationProfile)
   }
   cy.visit(Cypress.env("base_url"));
 
-  // Wait 5s for the events to start firing. In actuality it probably doesn't need to be
-  // this log but it also gives time for the handler above to fire as well.
-  cy.wait(5000);
-
   // Wait for any requests that were made to finish. We give all requests 1m. They
   // probably don't need that long, but they can have it.
   if (requestsMade.length) {
     cy.wait(requestsMade, { timeout: 60000 });
   }
-
-  cy.get("p")
+  
+  cy.get('ul')
+    .find('li', { timeout: 15000 })
+    .should('have.attr', 'aria-expanded', 'false') 
```

Comment:
Hmm is this going to be good enough? Are these the only `ul` and `li` that we hvave that could be chosen by this?

Reply:
If it gets the correct `ul`, the `li` should only be found within that list. The `ul` has a `aria-label="file system navigator"` attribute which would allow us to ensure we get the right list. It is not a terrible use of an `aria-label` I suppose.